probe/endpoint/reporter.go

```diff
@@ -64,12 +65,16 @@ func NewReporter(hostID, hostName string, includeProcesses bool, useConntrack bo
 			log.Printf("Failed to start natMapper: %v", err)
 		}
 	}
+
+	revRes := NewReverseResolver(rAddrCacheLen)
```

Comment:
I left the configurable cache length because an important part of the reverse resolver is the cache, and that is configurable with that parameter, so I was reluctant to hardcode that value with a constant... but it is not a big deal, I will use a constant.